app/javascript/src/components/Dashboard/Notes/Item.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import { Clock, MenuVertical } from "neetoicons";
+import { Typography, Tag, Dropdown, Avatar, Tooltip } from "neetoui";
+
+const Item = ({ note }) => (
+  <div className="m-3 rounded-sm border border-gray-200 p-4 text-gray-500 shadow-md">
+    <div className="flex justify-between">
+      <Typography style="h3">{note.title}</Typography>
+      <Dropdown buttonStyle="text" icon={MenuVertical}>
+        <li>Edit</li>
+        <li>Delete</li>
```

Comment:
Remember to apply the translations. If you are going to apply it in the future you may ignore the comment.